fixtures/groovy/admin/createSite.groovy

```diff
@@ -1,14 +1,35 @@
+import org.jahia.api.Constants
+import org.jahia.services.content.JCRCallback
+import org.jahia.services.content.JCRSessionWrapper
+import org.jahia.services.content.JCRTemplate
+import org.jahia.services.content.decorator.JCRSiteNode
 import org.jahia.services.sites.JahiaSite
 import org.jahia.services.sites.JahiaSitesService
 import org.jahia.services.sites.SiteCreationInfo
+import org.jahia.services.usermanager.JahiaUserManagerService
+
+import javax.jcr.RepositoryException
 
 JahiaSitesService sitesService = JahiaSitesService.getInstance();
 if (sitesService.getSiteByKey("SITEKEY") == null) {
     JahiaSite site = sitesService.addSite(SiteCreationInfo.builder().
             siteKey("SITEKEY").
-            serverName("localhost").
+            serverName("SERVERNAME").
             title("SITEKEY").
             templateSet("TEMPLATES_SET").
-            locale("en").build())
+            locale("LOCALE").build())
+
+    JCRCallback callback = new JCRCallback() {
```

Comment:
Maybe, you could use new JCRCallback() directly in doExecuteWithSystemSessionAsUser instead of creating a object here .
If you want, you can also replace with a lambda
Example:
```
JCRTemplate.instance.doExecuteWithSystemSessionAsUser(JahiaUserManagerService.instance.lookupUser("root").jahiaUser,
            Constants.EDIT_WORKSPACE, Locale.ENGLISH, jcrsession -> {
        JCRSiteNode siteByKey = sitesService.getSiteByKey("SITEKEY", jcrsession)
        siteByKey.setLanguages(new HashSet<String>(LANGUAGES))
        jcrsession.save()
        return null
    })
```

Reply:
Fixed, thanks